src/core/reaction.ts

```diff
@@ -105,7 +105,7 @@ export class Reaction<T>
    * @param another Reaction to compare this reaction's priority against.
    */
   hasPriorityOver(another: PrioritySetElement<Priority> | undefined): boolean {
-    if (another != null && this.getPriority() < another.getPriority()) {
+    if (another !== undefined && this.getPriority() < another.getPriority()) {
```

Comment:
We should revert it back, it appears that this is a caching issue and could be fixed by re-running `npx esline` or restarting TS server

Reply:
OK. Is the original code correct and the change wrong?

Reply:
The original code is correct yet the change is not wrong. It's just (I feel) a common practice to always use `!= null` instead of `!== undefined`, even if the variable is non-nullable.